<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialwindow</class>
 <widget class="QWidget" name="serialwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>11</y>
     <width>551</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>接收区：</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QTextBrowser" name="Serial_Receive_text_window"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>发送区：</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QTextEdit" name="Serial_send_text_window"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QPushButton" name="Serial_send_button">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>248</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="Serial_manysend_button">
             <property name="text">
              <string>多项发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>217</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QGroupBox" name="SerialConf">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>串口配置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="Serial_Label_PortBox">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Serial_PortBox">
             <item>
              <property name="text">
               <string>COM1</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="Serial_Label_BaudRateBox">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Serial_BaudRateBox">
             <property name="currentText">
              <string>1200</string>
             </property>
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>56000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>194000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1800000</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="Serial_Label_DataBitBox">
             <property name="text">
              <string>数据位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Serial_DataBitBox">
             <property name="currentText">
              <string>5</string>
             </property>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="Serial_Label_StopBitBox">
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Serial_StopBitBox">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="Serial_Label_CheckBitBox">
             <property name="text">
              <string>校验位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Serial_CheckBitBox">
             <item>
              <property name="text">
               <string>NONE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ODD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EVEN</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="Serial_Label_Open">
           <property name="styleSheet">
            <string notr="true">image: url(:/seialimage/close.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Serial_Open">
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>数据显示格式：</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QCheckBox" name="Serial_checkBoxHexS">
           <property name="text">
            <string>Hex发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Serial_checkBoxHexR">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Hex显示</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QCheckBox" name="Serial_checkBox_Continuous_send">
           <property name="text">
            <string>连续发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Serial_LineEdit_checkBox_Continuous_send">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QCheckBox" name="Serial_checkBox_frameline">
           <property name="text">
            <string>帧换行  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Serial_LineEdit_checkBox_frameline">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>40</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QPushButton" name="Serial_pushButtonclearsend">
           <property name="text">
            <string>清除发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Serial_pushButtonclearrecv">
           <property name="text">
            <string>清除接收</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Serial_jiankong">
         <property name="text">
          <string>串口协议监控</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Serial_count_rev">
         <property name="text">
          <string>重新计数</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
